src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/MemberRepository.cs

```diff
@@ -123,18 +207,21 @@ public async Task UpdateMemberAsync(Member member)
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"UPDATE Members 
-                                 SET Email = @Email, 
-                                     IsAfk = @IsAfk, 
-                                     IsBanned = @IsBanned, 
-                                     LastActivity = @LastActivity, 
-                                     LastNudged = @LastNudged, 
-                                     Name = @Name, 
-                                     PhotoName = @PhotoName,
-                                     Role = @Role, 
-                                     SaasUserId = @SaasUserId, 
-                                     Status = @Status
-                                 WHERE Id = @Id";
+                var sqlQuery = $@"
+                    UPDATE Members 
+                    SET 
+                        {nameof(Member.Email)} = @{nameof(Member.Email)},
+                        {nameof(Member.IsAfk)} = @{nameof(Member.IsAfk)}, 
+                        {nameof(Member.IsBanned)} = @{nameof(Member.IsBanned)}, 
+                        {nameof(Member.LastActivity)} = @{nameof(Member.LastActivity)}, 
+                        {nameof(Member.LastNudged)} = @{nameof(Member.LastNudged)}, 
+                        {nameof(Member.Name)} = @{nameof(Member.Name)}, 
+                        {nameof(Member.PhotoName)} = @{nameof(Member.PhotoName)},
+                        {nameof(Member.Role)} = @{nameof(Member.Role)}, 
+                        {nameof(Member.SaasUserId)} = @{nameof(Member.SaasUserId)}, 
+                        {nameof(Member.Status)} = @{nameof(Member.Status)}
```

Comment:
Should we set here property {nameof(Member.IsDeleted)} after merging new version.

Reply:
We won't forget :)